### DISCUSSION TITLE
Version 2.2.3 has issue with filtering

### DISCUSSION THREAD

**Discussion by ShivrajJoshi1996** [Budibase/budibase#9100] [Help] [answered]
Hi Budibase Team,

I was really working hard on Budibase for our internal project and it was almost done but last night budibase auto updated to the newer version and all the setting that I did is messed up. 

I tried to revert it back but no luck on it.

I tried to fix almost of it but one major issue that I am having is that earlier filtering was working good but now the condition that I have set for filter is not working and I spent whole day to figure it out but it still not filtering.

I don't know whether it is a bug or there is some other way of filtering data in newer version.

This is the example how earlier I was filtering data:

Let say I have two screens, Screen 1 and Screen 2 and for each screen I have two database table with one relationship column between these two.

On screen one I have selected a row from a table and click on the row to go to the Screen 2

On Screen 2 first I took data provider with the database table of Screen 1 and use the repeater and took the Text component and inside it used the column name by which I want to filter the data of Screen 2. 

Now from the Text component that I have (relationship column on Screen 1 database table and Screen 2 database table) I was filtering the Table component of Screen 2.

above scenario was completely working before this latest update but now its not.

Please need your help on this as soon as possible.

Thanks in advance,
Shivraj Joshi

**Comment by melohagan** [comment, marked as answer]
Bug raised and discussed here: https://github.com/Budibase/budibase/issues/9104
